src/wasm/wasm-validator.cpp

```diff
@@ -1914,7 +1914,7 @@ void FunctionValidator::visitTupleMake(TupleMake* curr) {
     }
     types.push_back(op->type);
   }
-  shouldBeTrue(Type(types) == curr->type,
+  shouldBeTrue(Type::isSubType(Type(types), curr->type),
```

Comment:
We have `shouldBeSubType` (and `shouldBeSubTypeOrFirstIsUnreachable` too as a counterpart of `shouldBeEqualOrFirstIsUnreachable`, even though this is not necessary here). The same for below.